website/versioned_docs/version-v2.2.0/gettingstarted/installation.mdx

```diff
@@ -65,6 +65,7 @@ import TabItem from "@theme/TabItem";
 ## Optional Dependencies
 
 - [UPX](https://upx.github.io/) for compressing your applications.
+- [NSIS](https://wails.io/docs/guides/windows-installer/) for generating Windows installers.
```

Comment:
I know it's annoying, but could we add this to `website/docs/gettingstarted/installation.mdx`? If you update a versioned doc, the changes will get lost. Thank you 👍

Reply:
Done